### DISCUSSION TITLE
im confused

### DISCUSSION THREAD

**Discussion by limitless1234** [ascii-boxes/boxes#117] [General]
it does not work the terminal is just doing nothing but i can still type and press enter and i put it all in the same folder but it still doesnt work :(
![Screenshot 2023-03-23 102750](https://user-images.githubusercontent.com/128695592/227175727-ebca672e-bea7-4644-9036-b7c17a872f76.png)

**Comment by tsjensen** [comment]
Well that's because you left stdin open. Learn more about standard stream here:
https://www.howtogeek.com/435903/what-are-stdin-stdout-and-stderr-on-linux/

Not an issue with boxes, though.
